src/foam/u2/grid/testdata/HeroModels.js

```diff
@@ -0,0 +1,213 @@
+foam.CLASS({
+    refines: 'foam.core.Property',
+    properties: [
+        {
+           name: 'referenceDAOKey',
+           documentation: 'get the property object from property Id through DAO', 
+        }, 
+        {
+            class: 'Function',
+            name: 'validator',
+        },
+        {
+            class: 'Function',
+            name: 'tableFormatter',
+            factory: function() { return function(value, obj, t){return value; }; },
+        },
+        {
+            class: 'Function',
+            name: 'gridHeaderView',
+            factory: function() { return function(value, obj, t){return value; }; },
+        },
+        {
+            name: 'sortable',
+            class: 'Boolean',
+            value: true, 
+        },
+        {
+            name: 'groupByProperty',
+            class: 'String',
+            value: 'id',
+        },
+        {
+            name: 'groupByLabel',
+            class: 'String',
+            value: 'label',
+        },
+        /*{
+            name: 'salesforceType',
+            class: 'String'
+        },*/
+    ],
+
+});
+
+
+
+// Relationship Test
+foam.CLASS({
+  name: 'Team',
+  requires: ['Hero'], 
+  properties: [ 'name', 'id' ]
+});
+
+foam.CLASS({
+  name: 'Hero',
+  
+  requires: ['Team'],
+  
+  imports: [
+    'TeamDAO', 
+  ], 
+  
+  properties: [
+    'id',
+    {
+        name: 'fieldworker',
+        referenceProperty: 'fieldworkerId',
+        referenceDAO: 'fieldworkerDAO', 
+    },
+    
+    {
+        name: 'name',
+        gridHeaderView: function(value, obj, t){
+            return value; 
+        },
+    },
+    {
+        class: 'Date', 
+        name: 'birthDate',
+    },
+    {
+        class: 'Date', 
+        name: 'lastSeenAlive',
+        searchView: 'foam.u2.search.WeekSearchView',
+        //searchView: 'foam.u2.DateView',  
+        factory: function(){
+            return SE.util.startOfToday();
```

Comment:
People are not going to like a global variable like 'SE'. com.something.util. something would be better.